crate/pages/blog/Snowsprint-2015.html

```diff
@@ -0,0 +1,75 @@
+title: Crate.IO Snowsprint 2015 recap: Dockerzeit
+link: https://crate.io/blog/Snowsprint-2015/
```

Comment:
you need to rename the file into lower case to achieve that. the `link` attribute has actually no effect